README.md

```diff
@@ -30,6 +30,41 @@ However, if you are looking to run a node a budget, you can use external HDD's s
 You'll need a SD card to flash the OS of choice (we will cover Raspbian and Manjaro-ARM).
 For this we recommend: [Samsung EVO+ 64GB](https://www.amazon.com/Samsung-MicroSDXC-Memory-Adapter-MB-MC64GA/dp/B06XFWPXYD/ref=sr_1_4?keywords=EVO%2B+SD+card&qid=1571081610&s=electronics&sr=1-4). You'll need at least 16GB but for the price this is a great deal for a high quality SD card. 
 
+## NFS Shares
+
+If you already have a NAS at home and don't want to buy another hard drive, there is always the possibility of serving bitcoin blocks over the local network on an NFS shared storage. We are asuming your `bitcoind` installation is external to Dojo.
```

Comment:
Adding these lines, and then creating a new document that I will like to on home page.